https://github.com/pytorch/pytorch/issues/26608
stop test using master version of torchvision, which is no longer compatible with PyTorch 1.2
Patch created by Simon Branford of the BEAR Software team at the University of Birmingham
--- test/test_utils.py.orig	2019-09-24 10:39:45.841517000 +0100
+++ test/test_utils.py	2019-09-24 10:40:26.006548891 +0100
@@ -532,7 +532,7 @@
 
     def test_load_from_github(self):
         hub_model = hub.load(
-            'pytorch/vision',
+            'pytorch/vision:v0.4.0',
             'resnet18',
             pretrained=True,
             progress=False)
@@ -543,17 +543,17 @@
         temp_dir = tempfile.gettempdir()
         hub.set_dir(temp_dir)
         hub_model = hub.load(
-            'pytorch/vision',
+            'pytorch/vision:v0.4.0',
             'resnet18',
             pretrained=True,
             progress=False)
         self.assertEqual(sum_of_model_parameters(hub_model),
                          SUM_OF_PRETRAINED_RESNET18_PARAMS)
-        assert os.path.exists(temp_dir + '/pytorch_vision_master')
-        shutil.rmtree(temp_dir + '/pytorch_vision_master')
+        assert os.path.exists(temp_dir + '/pytorch_vision_v0.4.0')
+        shutil.rmtree(temp_dir + '/pytorch_vision_v0.4.0')
 
     def test_list_entrypoints(self):
-        entry_lists = hub.list('pytorch/vision', force_reload=True)
+        entry_lists = hub.list('pytorch/vision:v0.4.0', force_reload=True)
         self.assertObjectIn('resnet18', entry_lists)
 
 if __name__ == '__main__':
